views/topic/edit-topic.hbs

```diff
@@ -44,10 +44,9 @@
 
                 <div class="form-group">
                     <label>{{$t "topic._topic.label.content" }}</label>
-
                     <!-- React Magic -->
-                    <div id="content-blocks" data-value="{{lesson.contents}}" data-parent-id="{{lesson._id}}" data-school-id="{{schoolId}}" data-parent-type="lessons" data-etherpadbaseurl="{{etherpadBaseUrl}}"
-                        data-iscoursegroup="{{courseGroupId}}" data-nexboardenabled="{{#ifEnv "FEATURE_NEXBOARD_ENABLED" "true"}}true{{/ifEnv}}"></div>
+                    <div id="content-blocks" data-testid="nextboardButton" data-value="{{lesson.contents}}" data-parent-id="{{lesson._id}}" data-school-id="{{schoolId}}" data-parent-type="lessons" data-etherpadbaseurl="{{etherpadBaseUrl}}"
+                        data-iscoursegroup="{{courseGroupId}}" data-nexboardenabled="{{#ifEnv "FEATURE_NEXBOARD_ENABLED" "false"}}true{{/ifEnv}}"></div>
```

Comment:
Isn't this "true"/"false" a default value for the given env var? If that's correct, then this will not change anything for the instances that have the `FEATURE_NEXBOARD_ENABLED` enabled anyway (which should stay set to "true" there as we want to keep the Nexboards usable until we'll have another solution working).